src/rust/engine/src/lib.rs

```diff
@@ -378,13 +378,11 @@ pub extern "C" fn scheduler_metrics(
 pub extern "C" fn scheduler_fork_context(
```

Comment:
Definitely a question for a separate PR, but I believe this is now unused, and it (and the python half which calls this) can also be deleted